content/en/docs/userguide/bypass.md

```diff
@@ -0,0 +1,158 @@
+---
+draft: false
+linktitle: Use Bypass Feature in Kmesh
+menu:
+  docs:
+    parent: user guide
+    weight: 21
+title: Use Bypass Feature in Kmesh
+toc: true
+type: docs
+---
+
+## **Overview**
+
+The bypass feature in Kmesh allows you to temporarily remove mesh service from the traffic path for troubleshooting purposes. This is particularly useful when you need to determine whether communication issues are caused by the mesh service or the application itself.
+
+## **Use Cases**
+
+###### 1. Troubleshooting Service Communication
+- Identify whether issues are caused by mesh service or application logic
+- Test direct communication between services without mesh interference
+- Debug network connectivity issues
+
+##### 2. Traffic Path Verification
+- Verify service behavior with and without mesh involvement
+- Validate application communication patterns
+- Test service-to-service direct connectivity
+
+## **Prerequisites**
+
+Before using the bypass feature, ensure you have:
+
+- A running Kubernetes cluster
+- Kmesh installed and configured (see [Quick Start Guide](https://kmesh.net/en/docs/setup/quickstart/))
+- Pods with sidecar injection enabled
+- Administrative access to the cluster
+
+## **Enabling Bypass**
+
+##### 1. Enable Bypass Controller
+
+The bypass controller must be enabled in your Kmesh configuration:
+
+```yaml
+--enable-bypass=true
```

Comment:
Please add more details, at least explain that it is added to the command line of Kmesh daemon.